src/structures/Guild.js

```diff
@@ -207,12 +207,42 @@ class Guild extends Base {
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
     /**
-     * The value set for a guild's default message notifications
+     * The value set for the guild's default message notifications
      * @type {DefaultMessageNotifications|number}
      */
     this.defaultMessageNotifications = DefaultMessageNotifications[data.default_message_notifications] ||
       data.default_message_notifications;
 
+    /**
+     * The maximum amount of members the guild can have
+     * @type {number}
+     */
+    this.maximumMembers = data.max_members;
```

Comment:
This field doesn't seem to be on the Guild Object within the GUILD_CREATE event.
```suggestion
    if (typeof data.max_members !== 'undefined') this.maximumMembers = data.max_members;
```
It would need to be retrieved from "fetching" the Guild via a GET request. Adding a Guild#fetch() that also calls `this._patch(data)` could be useful for developers that want to fill in the missing properties, something like this perhaps?
```js
/**
 * Fetch this guild.
 * @returns {Promise<Guild>}
 */
fetch() {
  return this.client.api.guilds(this.id).get().then(data => {
    this._patch(data)
    return this;
  });
}
```